bal/blockchain.py

```diff
@@ -119,47 +132,89 @@ def do_new_transaction():
 
     return jsonify(tx), 200
 
+
 @app.route('/transactions/has_amount/<int:amount>', methods=['GET'])
 def do_has_amount_for_transaction(amount):
     try:
-        create_transaction(get_public_from_wallet(), amount, get_private_from_wallet(), blockchain.get_unspent_tx_outs(), blockchain.transaction_pool.get_transaction_pool())
+        create_transaction(
+            get_public_from_wallet(),
+            amount,
+            get_private_from_wallet(),
+            blockchain.get_unspent_tx_outs(),
+            blockchain.transaction_pool.get_transaction_pool())
```

Comment:
Is it possible to group these variables by let's say 2-2? I mean 2 parameter for each line.